etcdctl/README.md

```diff
@@ -805,6 +805,29 @@ Prints a line of JSON encoding the database hash, revision, total keys, and size
 +----------+----------+------------+------------+
 ```
 
+### MOVE-LEADER \<hexadecimal-transferee-id\>
+
+MOVE-LEADER manually transfers a leadership to another node in the cluster.
```

Comment:
`MOVE-LEADER transfers leadership from the leader to another member in the cluster.`